src/components/callstoaction/CallToActionButton/__tests__/CallToActionButton-test.js

```diff
@@ -1,22 +1,148 @@
 "use strict";
 jest.autoMockOff();
 
+var React       = require('react/addons');
+var CallToActionButton      = require('../');
+var TestUtils   = React.addons.TestUtils;
+var findByClass = TestUtils.findRenderedDOMComponentWithClass;
+var scryByTag = TestUtils.scryRenderedDOMComponentsWithTag;
+
 describe('CallToActionButton', function() {
-  var React                       = require('react/addons');
-  var CallToActionButton          = require('../');
-  var TestUtils                   = React.addons.TestUtils;
+  var component;
+
+  describe('cta', function() {
+    var clicked = false;
+
+    beforeEach(function() {
+      clicked = false;
+      var onClick = function(){
+        clicked = true;
+      };
+
+      component = TestUtils.renderIntoDocument(
+        <CallToActionButton kind='cta' label='Get Started' icon='chevron-right' onClick={ onClick }/>
+      );
+    });
+
+    it('renders a CTA CallToActionButton', function(){
```

Comment:
Nitpick but "it renders a call to action call to action button" is odd (assuming just remnant from renaming component)